develop/docs/dlq.md

```diff
@@ -0,0 +1,35 @@
+# Managing DLQ Using tdbg
+When a task in the Temporal server encounters a terminal error, it may be enqueued in the Dead Letter Queue (DLQ). 
+To effectively manage these DLQ messages, follow these steps:
+
+1. There is a metric `dlq_writes`, which will be incremented each time a message is enqueued to the DLQ.
+2. Search `Task enqueued to DLQ` in logs and see task details. Get the DLQ message ID from this log line.
+3. From the log trace, you can identify the offending namespace, task type, source cluster, and target cluster of the task.
+4. From the task type, get the corresponding dlq type from this map:
+    1. transfer: 1
+    2. timer: 2
+    3. replication: 3
+    4. visibility: 4
+5. You can list the DLQ messages using the command
+   `tdbg dlq --dlq-version v2 read --dlq-type {type}`. Substitute `{type}` with the integer value from step 4.
```

Comment:
I'd also note that you can specify `--cluster` as the source cluster for replication tasks and you can specify `--max-message-id` to be the message ID you got from the logs.